src/utils/makeReactNativeConfig.js

```diff
@@ -97,6 +99,15 @@ const getDefaultConfig = ({
           // eslint-disable-next-line no-useless-escape
           exclude: /node_modules(?!.*[\/\\](react|@expo|pretty-format|haul|metro))/,
           use: [
+            {
+              loader: require.resolve('cache-loader'),
+            },
+            {
+              loader: require.resolve('thread-loader'),
+              options: {
+                workers: Math.max(os.cpus().length * 2 - 1, 1),
```

Comment:
why `os.cpus().length * 2 `?

Reply:
2 threads for 1 core, common recommendation for scaling

Reply:
`os.cpus()` will return number of all available cores (physical + logical), no need to blindly assume that we have twice as more available (e.g. on my MBP 2015 it would return 15 workers instead of 7, as I have 4 physical + 4 logical cores available). Please change to `Math.max(os.cpus().length - 1, 1) `

Reply:
done